vault/resource_identity_group_alias.go

```diff
@@ -49,6 +52,10 @@ func identityGroupAliasResource() *schema.Resource {
 }
 
 func identityGroupAliasCreate(d *schema.ResourceData, meta interface{}) error {
+	lock, unlock := getEntityLockFuncs(d, identityGroupAliasIDPath)
```

Comment:
nit: is it worth updating the function name to something more generic, like `getLockFuncs` since it is not specific to entities?

Reply:
I think I will keep it as-is since an Identity Group is a collection of Entities and the lock keys that we create are specific to entities in that the key is tied to the mount accessor. I don't see this function being used much or at all outside of entities use cases.